main.py

```diff
@@ -105,10 +105,10 @@ def debug_gql(self) -> int:
     parser.add_argument("--log", action="store_true")
     # undocumented debug args
```

Comment:
I use the VSCode debugger for all of my debugging needs, so this is mostly unused code. Still, as long as anyone finds this useful in any way.
```suggestion
    # debug options
```